frontend/src/components/CANs/CANTypes.d.ts

```diff
@@ -66,16 +66,18 @@ export type FundingBudget = {
 };
 
 export type FundingDetails = {
+    active_period?: number;
```

Comment:
@rajohnson90 you may want to check the schema since I don't see these additions in the OpenAPI spec